content/docs/command-reference/run.md

```diff
@@ -183,6 +183,9 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
   > Note that DVC-files without dependencies are automatically considered
   > "always changed", so this option has no effect in those cases.
 
+- `--external` - allow outputs that are outside of the DVC repository. See
+  [Managing External Data](/doc/user-guide/managing-external-data).
+
```

Comment:
But external dependencies don't require the use of this flag? Hmmm that may be confusing 😕

Reply:
Yep, only outs.

Reply:
Moved to https://github.com/iterative/dvc/issues/1545#issuecomment-647760462 now that this is merged. Thanks